Assets/HoloToolkit/UX/Scripts/AppBar/AppBarButton.cs

```diff
@@ -0,0 +1,184 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using HoloToolkit.Unity.Buttons;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.UX
```

Comment:
Please add summary and other comments to facilitate documentation generation.

Thanks!